securitycenter/muteconfigv2/mute_config_test.go

```diff
@@ -0,0 +1,285 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package muteconfigv2
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"time"
+
+	"io"
+	"log"
+	"os"
+	"strings"
+	"testing"
+
+	securitycenter "cloud.google.com/go/securitycenter/apiv2"
+	"cloud.google.com/go/securitycenter/apiv2/securitycenterpb"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+	"github.com/golang/protobuf/ptypes"	
+	"github.com/google/uuid"
+)
+
+var fixture *muteconfigFixture
+
+func TestMain(m *testing.M) {
+	var err error
+	fixture, err = newMuteConfigFixture()
+	if err != nil {
+		log.Fatalf("failed to create fixture: %s", err)
+	}
+
+	exitCode := m.Run()
+
+	if err := fixture.Cleanup(); err != nil {
+		log.Fatalf("failed to cleanup resources: %s", err)
+	}
+	fixture.client.Close()
+	os.Exit(exitCode)
+}
+
+func createFinding(findingID string, category string, sourceName string, orgId string) (*securitycenterpb.Finding, error) {
+	ctx := context.Background()
+	client, err := securitycenter.NewClient(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("securitycenter.NewClient: %v", err)
+	}
+
+	eventTime, err := ptypes.TimestampProto(time.Now())
+	if err != nil {
+		return nil, fmt.Errorf("TimestampProto: %v", err)
+	}
+
+	req := &securitycenterpb.CreateFindingRequest{
+		Parent:    sourceName,
+		FindingId: findingID,
+		Finding: &securitycenterpb.Finding{
+			State: securitycenterpb.Finding_ACTIVE,
+			// Resource the finding is associated with.  This is an
+			// example any resource identifier can be used.
+			ResourceName: fmt.Sprintf("//cloudresourcemanager.googleapis.com/organizations/%s", orgId),
+			// A free-form category.
+			Category: category,
+			// The time associated with discovering the issue.
+			EventTime: eventTime,
+			Severity:  securitycenterpb.Finding_LOW,
+		},
+	}
+	return client.CreateFinding(ctx, req)
+}
+
+func createSource(w io.Writer, orgId string) error {
+	ctx := context.Background()
+	client, err := securitycenter.NewClient(ctx)
+	if err != nil {
+		return fmt.Errorf("securitycenter.NewClient: %v", err)
```

Comment:
Nit: I think `fmt.Errorf` is always better with the verb `%w` for an error, even in a test file  (this is different from `t.Errorf`)

4 occurrences

Reply:
Addressed.